pysumma/simulation.py

```diff
@@ -64,22 +93,25 @@ def gen_summa_cmd(self, processes=1, prerun_cmds=[],
 
         return preprocess_cmd + summa_run_cmd
 
-    def run_local(self, run_suffix=None, processes=1, prerun_cmds=[],
-                  startGRU=None, countGRU=None, iHRU=None, freq_restart=None,
-                  progress=None):
-        run_cmd = self.gen_summa_cmd(processes, prerun_cmds,
-                                     startGRU, countGRU, iHRU, freq_restart,
-                                     progress)
+    def _run_local(self, run_suffix, processes=1, prerun_cmds=None,
+                   startGRU=None, countGRU=None, iHRU=None, freq_restart=None,
+                   progress=None):
+        """Start a local simulation"""
+        run_cmd = self._gen_summa_cmd(run_suffix, processes, prerun_cmds,
+                                      startGRU, countGRU, iHRU, freq_restart,
+                                      progress)
         self.process = subprocess.Popen(run_cmd, stdout=subprocess.PIPE,
                                         stderr=subprocess.PIPE, shell=True)
-        self._status = 'Running'
+        self.status = 'Running'
 
-    def run_docker(self, run_suffix=None, processes=1,
-                   prerun_cmds=[], startGRU=None, countGRU=None, iHRU=None,
-                   freq_restart=None, progress=None):
-        run_cmd = self.gen_summa_cmd(processes, prerun_cmds,
-                                     startGRU, countGRU, iHRU,
-                                     freq_restart, progress)
+    def _run_docker(self, run_suffix, processes=1,
+                    prerun_cmds=None, startGRU=None, countGRU=None, iHRU=None,
+                    freq_restart=None, progress=None):
+        """Start a docker simulation"""
+        run_cmd = self._gen_summa_cmd(run_suffix, processes, prerun_cmds,
+                                      startGRU, countGRU, iHRU,
+                                      freq_restart, progress)
+        run_cmd = run_cmd.replace(self.executable, '/code/bin/summa.exe')
 
         fman_dir = os.path.dirname(self.manager_path)
         settings_path = self.manager.settings_path.value
```

Comment:
self.manager doesn't have settings_path attribute.
So change to 
`settings_path = self.manager.get_value("settings_path")`
`input_path = self.manager.get_value("input_path")`
`output_path = self.manager.get_value("output_path")`

Reply:
This should now be fixed.